### DISCUSSION TITLE
Revolt not available in Russia without VPN

### DISCUSSION THREAD

**Discussion by Sh18RW** [stoatchat/discussions#880] [Q&A]
Revolt uses Cloudflare, which provides ECH. But this was blocked. I wrote to the support, where it was confirmed. The administration knows it. Two months have passed. There is nothing changed: Revolt still not work without VPN.
I am extremely interested in having a decent replacement for blocked Discord. Are there any plans to implement a mirror, for example, for Russian users? I believe Revolt is also interested in spreading the platform here. As practice has shown, in theory, russian users are ready to support and even develop it. Is there any plans for this, or is Revolt not interested in what I wrote?

**Comment by BarkovMA** [comment]
oh really? I just checked that my VPN is disabled, I logged into Revolt few mins ago. May this block depend on your ISP?

**Reply by ExtSeer** [reply]
It depends on the browser settings. Everything works fine when ECH is disabled, but it's difficult to explain this to the people I'm inviting.
